spec/validators/application_fulfilment_validator_spec.rb

```diff
@@ -10,13 +10,61 @@ module Test
 RSpec.describe ApplicationFulfilmentValidator, type: :model do
   subject { Test::C100ApplicationValidatable.new(arguments) }
 
-  let(:arguments) { { children: children, applicants: applicants, respondents: respondents } }
+  let(:arguments) do
+    {
+      children: children,
+      applicants: applicants,
+      respondents: respondents,
+      payment_type: payment_type,
+      submission_type: submission_type,
+    }
+  end
 
   let(:children)    { [Object] }
   let(:applicants)  { [Object] }
   let(:respondents) { [Object] }
+  let(:payment_type) { PaymentType::SELF_PAYMENT_CARD.to_s }
```

Comment:
I think is better not to couple these tests to any specifics, and not use value objects.
After all, we are only testing for presence, so anything will do.